.teamcity/_self/projects/WebApp.kt

```diff
@@ -20,8 +20,8 @@ object WebApp : Project({
 	buildType(RunAllUnitTests)
 	buildType(CheckCodeStyleBranch)
 	buildType(BuildDockerImage)
-	buildType(RunCalypsoPlaywrightE2eDesktopTests)
-	buildType(RunCalypsoPlaywrightE2eMobileTests)
+	buildType(playwrightBuildType("desktop", "23cc069f-59e5-4a63-a131-539fb55264e7", "+:*"));
+	buildType(playwrightBuildType("mobile", "90fbd6b7-fddb-4668-9ed0-b32598143616", "+:trunk"));
```

Comment:
Why only `trunk`? We should run this for branches ASAP, now that we have deleted the equivalent Selenium tests.

Reply:
I was playing it a bit safe, not wanting to overload the only 8 workers we have - but yeah, in that case I can eliminate the `filter` param and use the following filter for both mobile and desktop:
```
"""
+:*
-:pull*
"""
```